easybuild/easyconfigs/p/PyTorch/PyTorch-1.1.0-foss-2019a-Python-3.7.2.eb

```diff
@@ -0,0 +1,264 @@
+    '13100c3deed300bbf16f87d8af3539f432462bfef9d38f0c7e3e387dc2e88676',  # PeachPy-20180219.tar.gz
+    '98af442368142a05acc2130289b2a31036e43dbba6b5ec7626e9fb2464ddceae',  # QNNPACK-20181227.tar.gz
+    '0684fafb0e9c43ae080a218c02360d13880a0dff39af70ac02c59c2298854314',  # fbgemm-20190418.tar.gz
+    '0de43b6eaddd356f1d6cd164f73f37faf2f6c96fd684e1f7ea543ce49c1d144e',  # benchmark-20180606.tar.gz
+    'fd9e9f0d6ef407a162f5f283c52d3c271efc1092377274abe1e30dff83dbc1a4',  # eigen-20180908.tar.gz
+    'e12e096e986221d96d55185e870397b3717b625d6c5d155fbd9a7be314bed06a',  # gemmlowp-20180828.tar.gz
+    'd0e486754d7ec96546e29b8386c4f76b3714cadccc240290b9dad27efa8fa4b2',  # nccl-20181213.tar.gz
+    '7224b03af4acbc54525105bb42e3ecd75c66a3a5b47e8a725ab008467c4109f9',  # cub-20170829.tar.gz
+    '7af50156df76c984edf781711d759a8827db1bfef223467eaec1627aa9a0e1a1',  # ideep-20190411.tar.gz
+    'b444b4428adc4befc777d9a1af2fd9bbf5f5a2339913668357196c5133e9cc7b',  # foxi-20190424.tar.gz
+    'fc05493c1ba7937eb350c69ebcf7a0b82c420d840cfe06de84bdecfd1332bce6',  # zstd-20171107.tar.gz
+    '815b0397cd4be02a302eca4072362aee92fa3f3ad594f7d43f8838c0bd3043d8',  # mkl-dnn-20190315.tar.gz
+    'f4129843d5c2996419f96f10928edd02b2150998861a088dc7cfa1b6a058102a',  # mklml_lnx_2019.0.3.20190220.tgz
+    '569333e194d6a1d716972e17ad2ca1cd72e92a8845cee2f02ab2d5f541f013e9',  # PyTorch-1.0.1-fix-for-intel.patch
+]
+
+# PyTorch pulls in a bunch of submodules which don't have releases.
+# We download the submodule revisions from their repos.
+# determine commit of additional sources via https://github.com/pytorch/pytorch/tree/v1.1.0/third_party
+builddependencies = [('CMake', '3.13.3')]
+
+dependencies = [
+    ('Python', '3.7.2'),
+    ('SciPy-bundle', '2019.03'),
+    ('numactl', '2.0.12'),
+    ('PyYAML', '5.1'),
+    ('hypothesis', '4.23.4'),
+    ('Ninja', '1.9.0'),
+    ('GMP', '6.1.2'),
+    ('MPFR', '4.0.2'),
+    ('gflags', '2.2.2'),
+    ('glog', '0.4.0'),
+    ('psutil', '5.6.3'),
+    ('FFmpeg', '4.1.3'),
+]
+
+download_dep_fail = True
+
+# This environment is needed to prevent setup.py from invoking git to determine version.
+prebuildopts = 'PYTORCH_BUILD_VERSION=%(version)s PYTORCH_BUILD_NUMBER=1 VERBOSE=1 && '
+prebuildopts += 'LDFLAGS="$LDFLAGS -ldl" && '
+prebuildopts += 'export EXTRA_CAFFE2_CMAKE_FLAGS="-DASMJIT_SRC_DIR=%(builddir)s/third_party/asmjit" && '
+prebuildopts += 'export USE_FFMPEG=ON && '
+prebuildopts += 'export USE_FBGEMM=ON && '
+
+preinstallopts = prebuildopts
+
+runtest = 'export PYTHONPATH=%(builddir)s/%(namelower)s-%(version)s/build/lib.linux-x86_64-%(pyshortver)s:$PYTHONPATH '
+runtest += '&& export LD_LIBRARY_PATH=%(builddir)s/%(namelower)s-%(version)s/torch/lib64:$LD_LIBRARY_PATH '
+runtest += '&& python test/run_test.py --verbose'
+
+options = {'modulename': 'torch'}
+
+postinstallcmds = ['cp -a %(builddir)s/%(namelower)s-%(version)s/torch/lib64/* %(installdir)s/lib/']
+
```

Comment:
@smoors Don't we need a sanity check here?

Reply:
@verdurin good idea, I'll add it.